src/black/__init__.py

```diff
@@ -498,6 +514,25 @@ def target_version_option_callback(
     callback=read_pyproject_toml,
     help="Read configuration from FILE path.",
 )
+@click.option(
+    "--report-type",
+    type=click.Choice([v.name.lower() for v in ReportType]),
+    callback=report_option_callback,
+    default=DEFAULT_REPORT,
+    multiple=False,
+    help=(
+        "Option if the generated Report should be to the Console or a Junit XML File.\n"
+        "[default : console]"
+    ),
+)
+@click.option(
+    "--junitxml",
+    type=click.File(mode="w"),
+    help=(
+        "Option is needed for --report-type junit it tells where to store the XML"
+        " Output.\nIf directory is not present it will be created"
```

Comment:
Suggested wording: "Write junit XML output to FILE. If the directory does not exist, it will be created."

Reply:
Will be changed.